src/components/MenuItem.js

```diff
@@ -119,6 +119,7 @@ function MenuItem(props) {
                 StyleUtils.getButtonBackgroundColorStyle(getButtonState(props.focused || hovered, pressed, props.success, props.disabled, props.interactive), true),
                 (hovered || pressed) && props.hoverAndPressStyle,
                 ...(_.isArray(props.wrapperStyle) ? props.wrapperStyle : [props.wrapperStyle]),
+                props.disabled && styles.buttonOpacityDisabled,
```

Comment:
Reducing the opacity as requested here https://github.com/Expensify/App/issues/18517#issuecomment-1542164455